src/AuthModule/Components/ChangePassword/ChangePassword.tsx

```diff
@@ -0,0 +1,209 @@
+import { Visibility, VisibilityOff } from "@mui/icons-material";
+import {
+  TextField,
+  FormControl,
+  FormLabel,
+  Button,
+  InputAdornment,
+  IconButton,
+  CircularProgress,
+  createTheme,
+} from "@mui/material";
+import { useContext, useState } from "react";
+import { useForm } from "react-hook-form";
+import { AuthContext } from "../../../Context/Components/AuthContext";
+import axios from "axios";
+import { SubmitHandler } from "react-hook-form";
+import { toast } from "react-toastify";
+type FormValues = {
+  oldPassword: string;
+  newPassword: string;
+  confirmPassword: string;
+};
+type Props ={
+  handleCloseDialog : ()=>void
+}
+
+
+export default function ChangePassword({ handleCloseDialog }: Props) {
+  const theme = createTheme({
```

Comment:
It is recommended to have the theme in a top-level component such as the App component, to have access to the the same design system in all components.